test/stats/config/stats.test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+// eslint-disable-next-line node/no-unpublished-require
+const { run } = require('../../utils/test-utils');
+
+describe('stats flag with config', () => {
+    it('should compile without stats flag', () => {
+        const { stderr, stdout } = run(__dirname, []);
+        expect(stderr).toBeFalsy();
+        expect(stdout).toContain(`stats: 'normal'`);
```

Comment:
webpack 4 and 5 return stats differently , thats why CI is failing for webpack@next. For reference you can check how it's implemented in #1627